frontend/app/src/components/section-components/chat/channels-list-item.tsx

```diff
@@ -4,34 +4,45 @@ import {
   faCommentDots as faMessage,
 } from '@fortawesome/free-solid-svg-icons';
 import { Channel, channelType } from '../../global-components/interface';
-import { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
 
-function ChannelsListItem({ channelItem }: { channelItem: Channel }) {
-  const [isActive, setIsActive] = useState(false);
+interface ChannelsListProps {
+  channelItem: Channel;
+  currentlyActivated: string;
```

Comment:
Maybe the name here could be activeChannel | activeChannelId

Reply:
changed